server/src/main/scala/org/ensime/server/Server.scala

```diff
@@ -115,13 +117,23 @@ object Server extends AkkaBackCompat {
   Slf4jSetup.init()
 
   val log = LoggerFactory.getLogger("Server")
+  val XDG_CONFIG_HOME = " ~/.config/ensime-server"
```

Comment:
we want to look for files called `.ensime-server.conf` in the project root and then merge in (with lower priority) a `~/.ensime-server.conf`